pre_commit/runner.py

```diff
@@ -17,18 +17,19 @@ class Runner(object):
     repository under test.
     """
 
-    def __init__(self, git_root):
+    def __init__(self, git_root, hook_type='pre-commit'):
```

Comment:
I'd rather we not add another optional argument to this class and instead change the property below to a function that takes the hook type.

Or better yet, make two properties, one for pre_commit_path, one for pre_push_path